build.gradle

```diff
@@ -287,26 +287,36 @@ dokkaHtml.shouldRunAfter mkdocs
 // endregion
 
 for (publishedProject in publishedProjects) {
-    def project = project(publishedProject.projectPath)
+    def project1 = project(publishedProject.projectPath)
 
-    project.pluginManager.apply('java')
-    project.pluginManager.apply('maven-publish')
-    project.pluginManager.apply('signing')
+    project1.pluginManager.apply('java')
+    project1.pluginManager.apply('maven-publish')
+    project1.pluginManager.apply('signing')
```

Comment:
What's the reason for this rename?

Reply:
There was a name collision between the function `project`, in line 290 and the variable `project`. This causes in line 307 where the function `project` is required again.

Reply:
I presumed it was due to groovy. Another reason to switch to the kotlin DSL.